Planovac_Udalosti/Program.cs

```diff
@@ -107,15 +84,26 @@ static void Main(string[] args)
                 Console.WriteLine("The program is closing.");
                 return;
             }
-            /*--------nejsem si jista jak jinak to napsat aby se mi pri opakovanem spatnem vstupu nezobrazovalo Choose an option 1-4 a hned za tim aby mi nevyjela zakladni "nabidka"----*/
```

Comment:
Myslim, ze to tady nemusime komplikovat, staci uzivateli vratit chybovou hlasku a zacit od zacatku 🙂